tests/dummy/config/environment.js

```diff
@@ -23,6 +23,12 @@ module.exports = function(environment) {
     }
   };
 
+  ENV['ember-a11y-testing'] = {
+    componentOptions: {
+      turnAuditOff: true
```

Comment:
I turned off `ember-a11y-testing` in development because I ran into an error while testing window resize:

```
element-resize-detector.js:165 Uncaught Axe is already running. Use `await axe.run()` to wait for the previous run to finish before starting a new run.
```